README.md

```diff
@@ -19,6 +19,14 @@ To set this up for a particular template from a project generated using Fuse Int
 $ oc create -n openshift -f fis-image-streams.json
 $ oc create -n myproject -f quickstart-template.json
 ```
+
+If you want to add all template following:
```

Comment:
what do you think to write it:
If you want to add all templates, use the following: